src/Chapter2.hs

```diff
@@ -841,7 +863,10 @@ list.
 
 🕯 HINT: Use the 'cycle' function
 -}
-rotate = error "rotate: Not implemented!"
+rotate :: Int -> [a] -> [a]
+rotate _ [] = []
+rotate n xs | n < 1 = []
```

Comment:
On negative numbers it should return an empty list, however, if it is `0`, then it means only that the list don't rotate, so you need to return the list as it is 👍🏼